layouts/shortcodes/page/about.html

```diff
@@ -0,0 +1,99 @@
+<div id="prowered-by" class="row featured-section featured-section-prefix text-center">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-14 col-md-offset-5 col-sm-18 col-sm-offset-3">
+                <p class="small">OUR MISSION</p>
```

Comment:
We should use the uppercase class instead of using uppercase text (accessibility)